### PR TITLE
Upload Area animation

### PR DESCRIPTION
**Pull request by BeeMargarida**
| - | - |
| --- | --- |
| Issue | -- |
| Decisions | Animation when dragging items above the upload area. |
| Animated GIF | Jan-16-2020 11-16-48 |

### CODE REVIEW

**Comment by gcandal** [comment]
@BeeMargarida ask someone to review this :)

File: vue/components/ui/molecules/upload-area/upload-area.vue [hunk 1]
```diff
@@ -42,14 +45,67 @@
...
+
+.fade-in-enter,
+.fade-in-leave-to {
+    opacity: 0;
+    transform: translateY(-10px);
+}
+
+.upload-area .upload-area-container .button-icon {
+    opacity: 0;
+    position: absolute;
```
Comment (3rdvision) [inline vue/components/ui/molecules/upload-area/upload-area.vue]: Fix on hover effect

File: vue/components/ui/molecules/upload-area/upload-area.vue [hunk 2]
```diff
@@ -55,7 +60,59 @@
...
+    opacity: 0;
+    transform: translateY(-10px);
+}
+
+.upload-area .upload-area-container .button-icon {
+    opacity: 0;
+    pointer-events: none;
+    position: absolute;
+    top: 40px;
+    transform: none;
```
Comment (3rdvision) [inline vue/components/ui/molecules/upload-area/upload-area.vue]: still being used?

**Comment by joamag** [comment]
@BeeMargarida good job 👍

**Pull request merged**